### PR TITLE
Tweak for test cases

### PR DESCRIPTION
**Pull request by cosmo0920**
This is still WIP.

### CODE REVIEW

File: fluent-plugin-irc.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_runtime_dependency "fluentd"
+  s.add_runtime_dependency "fluentd", [">= 0.14.0", "< 2"]
```
Comment (okkez) [inline fluent-plugin-irc.gemspec]: How about omit `[]`?

```ruby
s.add_runtime_dependency "fluentd", ">= 0.14.0", "< 2"
```

Comment (cosmo0920) [inline fluent-plugin-irc.gemspec]: For simplicity of appearance?
I don't want to omit square brackets.
This is more easy-to-understand notation for me.

**Comment by cosmo0920** [comment]
@choplin Could you release this plugin as 0.1.0?
I've finished to migrate using v0.14 API.

**Comment by choplin** [comment]
Thank you guys! I've released the gem as v0.1.0.
I will add you to the gem owners so that you can release the gem by yourself, if you want.

BTW, I've fixed the homepage field in the gemfile https://github.com/fluent-plugins-nursery/fluent-plugin-irc/commit/0626b31b0b1fb09bab1962488a1433c6e12faf3f
Please feel free to edit other fields in the gemspec (e.g. authors or email).

**Comment by cosmo0920** [comment]
Thanks for releasing new version of this plugin!



> I will add you to the gem owners so that you can release the gem by yourself, if you want.

Yes, I do.
@okkez and I want to maintain and release by ourselves in the future.

**Comment by choplin** [comment]
Please tell me your and @okkez 's email addresses.

**Comment by cosmo0920** [comment]
@okkez address is [here](https://github.com/fluent-plugins-nursery/contact/issues/2#issuecomment-291084068).
And mine is `cosmo0920.oucc@gmail.com`

**Comment by choplin** [comment]
Sorry, I missed the mention at https://github.com/fluent-plugins-nursery/contact/issues/2.
I've added you and @okkez to the owners.

**Comment by okkez** [comment]
Thank you very much!

**Pull request merged**